js/curation-tool.js

```diff
@@ -476,6 +476,26 @@ const vm = new Vue({
       // We've set up a data-dismiss to do that, so we don't need to do
       // anything here, but the function is here in case we need it later.
     },
+    deleteTUnit(tunitToDelete) {
+      const tunit = tunitToDelete;
```

Comment:
I know there's a school of thought out there that says `const` doesn't mean immutable, it only means it's like `let` except I declare not to reassign the value. Is that what you're trying to follow?

I guess I'm OK with that, though the more traditional (outside of JS) connotation of `const` is immutable.

Reply:
Yup, this is how Airbnb's style guide likes to work [with complex types](https://github.com/airbnb/javascript#types--complex) -- note that the example they use has a "const" array whose values are changed! I'm fine with `const` meaning "reference doesn't change" rather than "object is immutable", but I'd be happy to try out the latter if you think it'd make the code more readable -- it looks like we'd just have to install and turn on [`eslint-plugin-const-immutable`](https://www.npmjs.com/package/eslint-plugin-const-immutable).

Reply:
Since you'll be the main programmer on the project, I'll leave the decision to you. If it were up to me, I'd use `const` in the meaning of immutable (rather than just not reassignable) and `let` otherwise. But I'm old school.

Note that the eslint plugin is only needed to actually enforce (in the sense of checking) immutability. Quite useful, but not strictly mandatory if you were concerned about the installation or runtime that's involved.

Reply:
I think using `const` to mean "cannot be reassigned" as compared to "immutable" makes sense within the language specification -- in JavaScript, you need to explicitly use [Object.freeze()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze) to make an object fully constant, and that [the specification refers only to the binding of the variable](https://mathiasbynens.be/notes/es6-const). So, I think we'll stick with `const` to mean only that the binding is immutable, not the variable itself, i.e. we'll leave [ESLint's `prefer-const` rule turned on](https://eslint.org/docs/rules/prefer-const) for now.

It also seems to be pretty ingrained within the JavaScript community (as evidenced by a quick survey of "const vs let" search results on Google and [the relevant post on StackExchange Software Engineering](https://softwareengineering.stackexchange.com/q/278652/122)). This will also make it easier for other JavaScript developers to join in on our project.